docs/content/workflows/design/hooks/kafka-hook.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Kafka hook"
+weight = 3
+
++++
+
+You want to run a workflow from a kafka message ? This kind of hook is for you.
+
+This kind of hook will connect to a kafka topic and consume message. For each message it will trigger your workflow.
```

Comment:
This kind of hook will connect to a kafka topic and consume message. For each message, it will trigger your workflow.

Reply:
messages

+ talk about payload

Reply:
done